api_mt/integrations/api_keys.md

```diff
@@ -0,0 +1,141 @@
+
+10. Type in `aws configure` again. This is an example of what you should see then:  
+
+![How to obtain an API key for Amazon Translate AWSpic4](trados/trados_for_amazon7.png)  
+
+You have successfully set up your AWS credentials, incl. your API key.  
+
+
+## 4.2 DeepL  
+
+
+Create a DeepL account [here](https://www.deepl.com/pro.html)  
+
+
+1. Select DeepL Pro payment plan. You need your debit/credit card to register your account. Provide your personal details. Confirm each step.  
+
+ 
+2. Press **Retrieve your Authentication Key** on the last screen. 
+
+
+3. Go to **Account**. Your API key is at the bottom of the screen under ** Authentication Key for DeepL API**. Copy it and store it in a safe location.  
+
+
+## 4.3 ModernMT  
+
+
+Create a ModernMT account [here](https://www.modernmt.com/pricing/).  
+
+1. Click **Plugins for translators** > **Get a license**.  
+
+
+2. You need your debit/credit card to register your account. Provide your personal details.  
+
+
+3. Click **Start your plan**.  
+
+
+4. Copy your activation key from the **Here’s your activation key** field. Store it in a safe location.  
+
+
+## Google Cloud Translation Basic (or Google Translate API v2)  
+
+
+To use this MT vendor you need to have your regular Google Account. Set it up and log in to it. Then log in [here](https://console.developers.google.com/cloud-resource-manager).
+
+1. Go to **Billing** > **Add a billing account**. Accomplish all the required steps of the procedure.  
+ 
+![How to obtain an API key_Google_pic1](trados/trados_for_google5.png)  
+
+2. You should find yourself in the **Dashboard** view. As there are frequent changes to the **Google Cloud Platform** the page you find yourself may be different.  
+
+![How to obtain an API key_Google_pic2](trados/trados_for_google6.png)  
+
+3. Type in `new project` in the search window. Click **Create a Project IAM & Admin**.  
+ 
+![How to obtain an API key_Google_pic3](trados/trados_for_google7.png)  
+
+4. Assign a name to your project in the **Project name** field (for example **CATpluginproject**). Write it down for future use. Click **Create**.  
+
+![How to obtain an API key_Google_pic4](trados/trados_for_google8.png)  
+
+5. Type in your project name in the search window and press **Enter**. Click the project name in the results window.  
+
+6. In the upper, left corner of the page, the drop-down menu has changed its name to the name of your project. Open the menu, and click the name of this project (for example **CATpluginproject**).  
+
+ ![How to obtain an API key_Google_pic5](trados/trados_for_google9.png)  
+
+7. The name of your project is now visible in the **Search** window in the middle of the upper part of the page. Click **Menu** (three horizontal lines one under another in the left corner) > **APIs & services**.  
+
+ ![How to obtain an API key_Google_pic6](trados/trados_for_google10.png)
+
+8. Click **ENABLE APIS AND SERVICES**.  
```

Comment:
I would keep it title case, even if it's in all caps in the app.

Reply:
I believe the instruction should match exactly what we see on the screen :)